TODO.md

```diff
@@ -15,10 +6,36 @@
 6. [x] click on download button, get the app in the chat, the download button changes to "downloading.." and
 7. [x] When the app was delivered, the state changes to "in chat"
 
-## Make reviw happy path run:
+## Make review happy path run:
 1. [x] User sends webxdc into 1:1 chat with the bot.
 2. [x] The bot creates a new group with the user where he forwards the webxdc to. It also send the helper wbxdc where things like name, description and so on can be seen and edited.
 3. [x] When all required fields are filled in, the user can send the webxdc to review.
 4. [x] Upon review request (send from helper xdc), the bot creates a new group with some testers and one publisher.
 5. [x] The testers test the app and checkmark required tests in the helper webxdc.
 6. [x] When all requirements are met, the publisher can publish the app to the store. 
```

Comment:
this was never discussed like this but i take it that's what is currently implemented -- for now, i am not commenting about it other than to say i think it's too many roles and everything could be based on the base/genesis store-admin group and anybody helping becomes part of this group.

Reply:
We talked about it at some point which gave me a general idea for this implementation. But yes, this is only an overview of what's implemented now and not part of the mvp.